metal/config.go

```diff
@@ -49,14 +51,13 @@ func MetalRetryPolicy(ctx context.Context, resp *http.Response, err error) (bool
 
 // Client returns a new client for accessing Equinix Metal's API.
 func (c *Config) Client() *packngo.Client {
-	httpClient := retryablehttp.NewClient()
-	httpClient.RetryWaitMin = time.Second
-	httpClient.RetryWaitMax = 30 * time.Second
-	httpClient.RetryMax = 10
-	httpClient.CheckRetry = MetalRetryPolicy
-	httpClient.HTTPClient.Transport = logging.NewTransport(
-		"Equinix Metal",
-		httpClient.HTTPClient.Transport)
-
+	transport := logging.NewTransport("Equinix Metal", http.DefaultTransport)
+	retryClient := retryablehttp.NewClient()
+	retryClient.HTTPClient.Transport = transport
+	retryClient.RetryMax = c.MaxRetries
+	retryClient.RetryWaitMin = time.Second
+	retryClient.RetryWaitMax = c.MaxRetryWait
+	retryClient.CheckRetry = packngo.RetryPolicy
```

Comment:
Let's continue to use `MetalRetryPolicy` (unless it is identical to the `packngo` copy, then perhaps we can remove the duplication here).

Reply:
The policy in pakcngo and the policy in the provider are the same. I would use the policy funciton from the provider. The policy function in packngo can be removed?